projects/cli.github.com/package.yml

```diff
@@ -20,6 +20,8 @@ build:
   dependencies:
     go.dev: ^1.18
     tea.xyz/gx/make: '*'
+    linux:
+      tea.xyz/gx/cc: c99 # for cgo
```

Comment:
oh ok, go.dev used to depend on this, but we made it a companion. Strictly this is not a linux dep, macOS needs it too. gx/cc on Mac checks if the Xcode CLT is installed.

Reply:
17922c85cfb7ca65b81dc7247a8f16b101f89906